src/java_tools/import_deps_checker/java/com/google/devtools/build/importdeps/DepsCheckerClassVisitor.java

```diff
@@ -58,7 +58,9 @@ public void visit(
       String[] interfaces) {
     checkState(internalName == null, "Cannot reuse this class visitor %s", getClass());
     this.internalName = name;
-    checkInternalName(superName);
+    if (superName != null) {
```

Comment:
could you possibly add a test that exercises this path, that is, build or provide a simple Jar file containing a module-info.class and run it through the tool?

Reply:
Sorry for the delay. I've added a test and I'm waiting for CI's results.